python/lsst/skymap/detail/dodecahedron.py

```diff
@@ -44,23 +49,38 @@ def __init__(self, withFacesOnPoles=False):
     def getFaceCtrList(self):
         """Return a list of face centers
 
-        @return a list of face centers (in index order); each a unit vector (numpy array)
+        Returns
+        -------
+        results : `self.faceVecList[:]`
```

Comment:
that's not a type

Reply:
Does this also just count as "callable"?

Reply:
I think it's a `np.ndarray`.